logprep/connector/opensearch/output.py

```diff
@@ -124,7 +131,7 @@ def describe(self) -> str:
     def _bulk(self, client, actions, *args, **kwargs):
         try:
             if self._config.parallel_bulk:
-                self._parallel_bulk(client, actions, *args, **kwargs)
+                self._parallel_bulk_with_retries(client, actions, *args, **kwargs)
                 return
```

Comment:
how about extending the signature of this `_bulk` method with `max_retries` like in the eleasticsearch implementation?